dist/t/spec/features/0030_project_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+RSpec.describe "Project" do
+  it "should be able to create" do
+    login('Admin','opensuse')
```

Comment:
I would put this line in a `before` block. And also remove `login` sentence in line 15.